addons/mail/static/src/js/followers.js

```diff
@@ -242,7 +242,8 @@ var Followers = AbstractField.extend({
     _unfollow: function (ids) {
         var self = this;
         var def = $.Deferred();
-        var text = _t("Warning! \n If you remove a follower, he won't be notified of any email or discussion on this document.\n Do you really want to remove this follower ?");
+        var follower = _.find(this.followers, {'res_id': (ids.partner_ids) ? ids.partner_ids[0] : ids.channel_ids[0]});
+        var text = _.str.sprintf(_t("Warning! \n If you remove a follower, he won't be notified of any email or discussion on this document.\n Do you really want to remove %s?"), follower && follower.name);
```

Comment:
Simply interpolate `follower.name` in string, since the field name is always required for partners and channels. Also, there should always a follower that matches the `_.find()` above.